demo_nodes_py/setup.py

```diff
@@ -1,17 +1,27 @@
-from ament_python.data_files import get_data_files
-from ament_python.script_dir import install_scripts_to_libexec
+import sys
+
 from setuptools import find_packages
 from setuptools import setup
 
 package_name = 'demo_nodes_py'
-data_files = get_data_files(package_name)
-install_scripts_to_libexec(package_name)
+
+# install scripts into libexec path
+if 'develop' in sys.argv:
+    sys.argv[sys.argv.index('develop') + 1:sys.argv.index('develop') + 1] = \
```

Comment:
This appears to insert the arguments but I don't see how it replaces a script-dir argument already part of argv the way that the ament_python version of this code did. https://github.com/ament/ament_python/blob/eafd268376cda400f68b1b5b8429986531b856ad/ament_python/script_dir.py#L27-L29

Same for the install command just below and the install-scripts argument which we do specify during debian packaging in order to make sure the non-libexec scripts are placed into `/opt/ros/$ROSDISTRO/bin`